ci-operator/config/backube/volsync/backube-volsync-main.yaml

```
@@ -32,6 +98,29 @@ tests:
   commands: ./.ci-scripts/pre-commit.sh
   container:
     from: src
+- as: e2e-openshift
+  optional: true
+  steps:
+    cluster_profile: aws
+    test:
+    - as: e2e
+      commands: |
+        # TODO: optimize this
+        set -x
+        KUBECONFIG="${SHARED_DIR}/kubeconfig"
+        export KUBECONFIG
+        cp $(echo "${KUBECONFIG}") /go/src/github.com/backube/volsync/test-kuttl/kubeconfig
+        helm repo add bitnami https://charts.bitnami.com/bitnami
+        helm repo update
+        helm install --create-namespace -n minio --set accessKey.password=access --set secretKey.password=password --set defaultBuckets=mybucket --set containerSecurityContext.enabled=false --set podSecurityContext.enabled=false minio bitnami/minio
```

In testing the last few days, it appears that recent bitnami/minio chart versions are broken on openshift.

```suggestion
        helm install --create-namespace -n minio --set accessKey.password=access --set secretKey.password=password --set defaultBuckets=mybucket --set containerSecurityContext.enabled=false --set podSecurityContext.enabled=false --version 7.3.0 minio bitnami/minio
```

---

pulled that in, thanks!

---

I think backube/volsync#79 is also going to be necessary to get it to pass consistently.